src/auth/basic/Config.cc

```diff
@@ -148,8 +149,106 @@ authenticateBasicStats(StoreEntry * sentry)
         basicauthenticators->packStatsInto(sentry, "Basic Authenticator Statistics");
 }
 
+inline size_t
+inlineUTF8SequenceLengthNonASCII(char b0)
```

Comment:
Please avoid specifying inlining preference in the function name. Focus on what the function does. BTW, it would be nice to add brief descriptions to these new helper functions.